lib/grape/formatters/active_model_serializers.rb

```diff
@@ -0,0 +1,15 @@
+# A grape response formatter that can be used as 'formatter :json, Grape::Formatters::ActiveModelSerializers'
```

Comment:
super small nitpick: If you're already making changes, would be good to capitalize `grape` here and elsewhere